src/components/ActualStepDescription/styled.ts

```diff
@@ -6,18 +6,20 @@ import { OverridableComponent } from "@material-ui/core/OverridableComponent";
 import { SvgIconTypeMap } from "@material-ui/core/SvgIcon";
 
 export const Title = styled.span`
-  font-size: 20px;
+  font-size: 130%;
   color: #36454c;
   font-weight: 800;
   display: block;
   text-align: center;
+  margin: 10px;
 `;
 
 export const Text = styled.span`
-  font-size: 12px;
+  font-size: 100%;
   color: #36454c;
   font-weight: 300;
   display: block;
+  margin: 10px;
```

Comment:
Wydaje mi się, że tak jest ładniej (jest taki sam rozmiar czcionki jak w teorii), ale jak coś to można wrócić ;)